hw/ip/sram_ctrl/dv/cov/sram_ctrl_cov_excl.el

```diff
@@ -0,0 +1,50 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Wed May  5 10:09:25 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[0].u_prim_alert_sender_parity.i_decode_ping
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[0].u_prim_alert_sender_parity.i_decode_ack
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[1].u_prim_alert_sender_parity.i_decode_ping
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[1].u_prim_alert_sender_parity.i_decode_ack
```

Comment:
IMO `prim_alert_sender` should be excluded in the cover cfg and we should re-enable coverage on its ports. We should check with Will Chen to see if we can do a wild-card in the cfg file to make it transparent to all parameter sets of alert sender.

Reply:
Super sorry for the delay. This PR https://github.com/lowRISC/opentitan/pull/6559 should exclude all prim_alert_sender except the toggle coverage.

Reply:
Thanks a lot for adding this! I can extend this to exclude all prince/present instances in a later PR.